app/models/katello/glue/candlepin/owner.rb

```diff
@@ -38,6 +38,32 @@ def content_access_mode
         self.owner_details['contentAccessMode']
       end
 
+      def enabled_product_content(for_repositories)
+        return [] if for_repositories && for_repositories.empty?
+        repositories = for_repositories.select([:product_id, :content_id]).to_a
+        content_ids = repositories.map(&:content_id)
+
+        filtered_product_content do |pc|
+          content_ids.include?(pc.content.id) && pc.product.enabled?
+        end
+      end
+
+      def filtered_product_content
+        cp_products = Katello::Resources::Candlepin::Product.all(Organization.first.label, [:id, :productContent])
```

Comment:
I dont remember exactly how this works. Can you enlighten me on why you are getting all products from Organization.first  as opposed to current org?

Reply:
that should be self ;)   /me thinks i copy and pasted from my console